src/styles/card.scss

```diff
@@ -0,0 +1,159 @@
+.catalog {
+  margin: auto;
+  padding: 50px 40px 74px;
+  max-width: $max-width;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  grid-gap: 48px;
+}
+
+@media (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 1fr);
+    justify-items: center;
+  }
+}
+
+@media (max-width: 767px) {
+  .catalog {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $active-color;
+}
+
+.review {
+  margin: 0;
+  font-size: 10px;
+  font-weight: $font-weight;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
it's allowed to move only this rule to `*` selector

Reply:
Not resolved. Move it to * selector and remove it from all other places.